src/transaction/makeTransferTransaction.js

```diff
@@ -2,6 +2,32 @@ import makeInputTemplate from './makeInputTemplate'
 import makeTransaction from './makeTransaction'
 
 
+// TODO: Can we remove `export` here somehow, but still be able to import the
+// function for tests?
+export function _makeTransferTransaction(
```

Comment:
@libscott and I created: https://github.com/bigchaindb/js-bigchaindb-driver/issues/44